base/src/main/java/com/smartdevicelink/managers/screen/menu/MenuCell.java

```diff
@@ -111,6 +121,29 @@ public MenuCell(@NonNull String title, @Nullable SdlArtwork icon, @Nullable List
         setParentCellId(MAX_ID);
     }
 
+    /**
+     * Creates a new MenuCell Object with multiple parameters and optional fields set
+     *
+     * @param title             The cell's primary text
+     * @param icon              The cell's image
+     * @param secondaryText     The cell's secondary text
+     * @param tertiaryText      The cell's tertiary text
+     * @param secondaryArtwork  The cell's secondary image
+     * @param voiceCommands     Voice commands that will activate the menu cell
+     * @param listener          Calls the code that will be run when the menu cell is selected
+     */
+    public MenuCell(@NonNull String title, @Nullable SdlArtwork icon, @Nullable String secondaryText, @Nullable String tertiaryText, @Nullable SdlArtwork secondaryArtwork, @Nullable List<String> voiceCommands, @Nullable MenuSelectionListener listener) {
```

Comment:
I suggest reordering the function params in both new constructors so the texts come after each other, and the images come after each other:
```
title, secondaryText, tertiaryText, icon, secondaryArtwork
```